docs/sources/setup-grafana/configure-grafana/proxy.md

```diff
@@ -0,0 +1,50 @@
+---
+aliases:
+  - ../features/datasources/proxy/
+description: Proxy datasource connections through a secure socks proxy.
+keywords:
+  - proxy
+  - guide
+title: Proxying data source connections
+weight: 1110
+---
```

Comment:
@stephaniehingtgen , general comment - under the configure-grafana directory, you should create a `proxy` subdirectory and within that directory, add an `index.md` file. The index.md file contains the content.